src/network/networkd-dhcp4.c

```diff
@@ -416,6 +482,8 @@ static int link_set_dhcp_routes(Link *link) {
                 }
         }
 
+        (void) dhcp_set_routing_policy_rules(link);
```

Comment:
Please move this to `dhcp4_address_ready_callback()` and do not ignore the failure.